images/win/scripts/Tests/MSYS2.Tests.ps1

```diff
@@ -0,0 +1,20 @@
+$env:msys2BinDir = "C:\msys64\usr\bin"
```

Comment:
I don't think we should use approach with `$env`. May be we may put those paths right to test? Or pass them via test cases